File: Python Excel Examples/workbook/get_workbook.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
configuration = ExcelConfiguration(appId, appKey)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

#input file format:  ods, xls, xlsx, xlsb, xlsm, xltm, xltx
name = "test.xltx"
password = ""
storage = ""
inputFolder = "input/data"
result = api.get_workbook(name, password=password, storage=storage, folder=inputFolder)

File: JavaScript Excel Examples/WorkbooksApiDemo/getWorkbook.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel= require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorkbookApi(apiClient);

    var name = "Workbook.xlsx";
    var opts = {folder : "input"};
    instance.getWorkbook(name, opts, 
        function (error, data, response) {
            if (error) throw error;
        });
})();